backend/scripts/demo_topics.sh

```
@@ -0,0 +1,68 @@
+NEW_ID=$(node -e 'const fs=require("fs");try{const j=JSON.parse(fs.readFileSync("/tmp/topic_create.json","utf8"));console.log((j.data&&j.data.topic&&j.data.topic.id)||"")}catch(e){console.log("")}')
+if [[ -n "${NEW_ID}" ]]; then
+  curl -s -X PUT "${BASE_URL}/api/v1/topics/${NEW_ID}" "${AUTH_HEADER[@]}" -H 'Content-Type: application/json' \
+    --data '{"description":"Office and remote work conversations"}' | jq . >/tmp/topic_update.json || true
+  curl -s -X DELETE "${BASE_URL}/api/v1/topics/${NEW_ID}" "${AUTH_HEADER[@]}" | jq . >/tmp/topic_delete.json || true
+fi
+
+echo "[8/8] Show database evidence (topics rows) via docker psql..."
+docker exec -t "${DB_CONTAINER}" psql -U postgres -d aurora_db -c 'SELECT id, name, category, englishlevel, prompts FROM "Topic" ORDER BY "createdAt" DESC LIMIT 5;' || true
+
+echo "[BONUS] Generate a chat prompt from first topic..."
+node -e "const {PrismaClient}=require('@prisma/client');const ChatService=require('./build/src/services/chat.service').default;(async()=>{const p=new PrismaClient();const t=await p.topic.findFirst();console.log('Topic:',t?.name);console.log('Prompt:\n',ChatService.buildPromptFromTopic(t));await p.$disconnect();})();"
```

Shellcheck (0.10.0)

[warning] disconnect is referenced but not assigned.

(SC2154)